src/App.js

```diff
@@ -13,10 +13,10 @@ export default function App() {
       <Header />
       <Switch>
         <Route exact path="/">
-          <GuideIntroduction />
+          <Home />
         </Route>
-        <Route path="/FirstTask">
-          <FirstTask />
+        <Route path="/task/:taskId">
```

Comment:
Does this go into a JSON object "task" and then add the taskId afterwards for the URL?

Reply:
And do you want to display a particular tasks taskId in the URL when you're loading the general list here?

Reply:
Question 1:  yes it references the id after the URL. It´s the route for the dynamic routing of the tasklist. 

Question 2: Based on that route, it will render each ID via dynamic routing so that each task will show up on a separate page/screen. The magic happens in the task list component code.